policy/policy.go

```diff
@@ -23,7 +23,6 @@ type repoConfig struct {
 	PCRepo         string
 	DHRepo         string
 	CSRepo         string
-	PCPrivate      bool
```

Comment:
Are there no private repos any more?

Reply:
Doesn't look like it. At least not the ones we manage by templates. From the config it was previously set only for mdcb, which is all public now.